src/agent/onefuzz/src/heartbeat.rs

```diff
@@ -93,7 +95,7 @@ where
 
         let flush_context = context.clone();
         let heartbeat_process = task::spawn(async move {
-            random_delay(heartbeat_period).await;
+            random_delay(initial_delay).await;
```

Comment:
should the initial delay still be random not that it is explicitly specified ? I think a jitter here is more appropriate here

Reply:
Normally, yes.  However, this is a user-initiated task, not something like launching 3000 tasks from CICD.  We shouldn't see a thundering herd here.